utility/flight_stow.py

```diff
@@ -32,26 +55,36 @@ def point_ants(ants, dry_run=False):
 	except:
 	    not_locked = []
 	    not_locked = [ant.name for ant in ants if ant.sensor.lock.get_value() == False]
-	    user_logger.warn('{} did not reach target'.format(not_locked))
-	time.sleep(5)
+	    user_logger.warn('{} did not reach target'.format(', '.join(not_locked)))
+	time.sleep(3)
 	ants.req.mode('STOP')
 
 # Set up standard script options
 usage = '%prog'
 description = 'receptor flight stow'
 parser = standard_script_options(usage, description)
 # Add experiment-specific options
+parser.add_option('--ridx-pos',
+                  type="string",
+                  default='x',
+                  help="move receiver indexers to specified position. default=%default")
+parser.add_option('--move-indexer',
+                  action="store_true",
+                  default= False,
```

Comment:
Should be `default=False` (pep8)

Reply:
Thanks, I'll make a note of the pep8 standard